.github/workflows/github-token-rotation.yml

```
@@ -0,0 +1,47 @@
+          # Create an installation access token
+          token_response=$(curl -s -X POST \
+            -H "Authorization: Bearer $jwt" \
+            -H "Accept: application/vnd.github.v3+json" \
+            https://api.github.com/app/installations/$installation_id/access_tokens)
+
+          NEW_TOKEN=$(echo $token_response | jq -r '.token')
+
+          # Update AWS Secrets Manager with the new token
+          aws secretsmanager put-secret-value --secret-id "$SECRET_NAME" --secret-string "$NEW_TOKEN"
+
+          echo "GitHub token rotated and updated in AWS Secrets Manager."
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)